src/Components/SavedSearchAlert/Utils/savedSearchCriteria.tsx

```diff
@@ -30,7 +30,11 @@ export const getAllowedSearchCriteria = (
     const isAllowedKey = allowedSearchCriteriaKeys.includes(key)
 
     if (isAllowedKey && !isDefaultValue(key, value)) {
-      allowedCriteria[key] = criteria[key]
+      if (Array.isArray(value)) {
+        allowedCriteria[key] = criteria[key].filter(v => v)
+      } else {
+        allowedCriteria[key] = criteria[key]
+      }
```

Comment:
**walkthrough**: I was running into some GraphQL errors where we'd fire a request with `attributionClass: [null]`.

This was on the artwork page where the artwork was missing an `attributionClass` value altogether, which occasionally is the case with artworks uploaded for inclusion in an auction.